src/process.rs

```diff
@@ -17,3 +17,18 @@ pub(crate) fn mark_as_not_cloexec(file: &impl AsFd) -> Result<()> {
 	.wrap_err("failed to set CLOEXEC on file")?;
 	Ok(())
 }
+
+pub(crate) fn mark_as_cloexec(file: &impl AsFd) -> Result<()> {
```

Comment:
This function isn't used now. If that is handled in launch-pad after forking, there should be no need for it here in the future.